exercises/bank-account/example.py

```diff
@@ -0,0 +1,44 @@
+import threading
+
+
+class BankAccount(object):
+    def __init__(self):
+        self.lock = threading.Lock()
+
+    def getBalance(self):
+        self.lock.acquire()
+        if (self.open is True):
+            self.lock.release()
+            return self.balance
+        else:
+            self.lock.release()
+            raise Exception
```

Comment:
Could it you please throw a narrower Exception, or at least state the root cause by:
```python
raise Exception('Describe what went wrong') 
```

Reply:
Perhaps [`ValueError`](https://docs.python.org/2/library/exceptions.html#exceptions.ValueError) would be better?